app/models/record/poi_record.rb

```diff
@@ -401,14 +401,7 @@ def parse_prices(xml_part)
     # Bei Auswahl der Kategorien „other“ und „discount“ wird hier die Kategorie frei benannt:
     # category wird dann gesetzt durch den Wert von categorytext
     def category_name_for_price(price)
-      xml_cat_name = price.at_xpath("category").try(:text)
-
-      case xml_cat_name
-      when "discount", "other"
-        price.at_xpath("categorytext").try(:text)
-      else
-        xml_cat_name
-      end
+      xml_cat_name = Translate.price_category(price)
```

Comment:
❌Hier ist `Translate` aber die Klasse heißt `Translation`.
Hast du das auch mal in der Konsole ausprobiert? Wenn nicht, bitte beim nächsten mal bedenken!

Reply:
Und die `xml_cat_name` braucht es hier nicht mehr oder? Könnte ja direkt returned werden.

Reply:
@donni106 Translate  war ein einfacher Tippfehler. Muss man nicht so drauf rumreiten finde ich.